core/stores/builder/builder.go

```diff
@@ -10,7 +10,7 @@ const dbTag = "db"
 
 // RawFieldNames converts golang struct field into slice string.
 func RawFieldNames(in interface{}, postgresSql ...bool) []string {
-	out := make([]string, 0)
+	var out []string
```

Comment:
It's not the same, please make sure `nil` and `[]`, which one is correct.

Reply:
empty slice use `var a []int`, not use `a := make([]int,0)`

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Hello, I don't see any references to this method here. From the test, it seems that there is no problem in using `var out []string`